### ISSUE TITLE
Batch script stdout no longer prints the path to cpl/cesm log file on failure

### ISSUE THREAD

**Issue by quantheory** [CESM-Development/cime#89]
As of beta06 (with updated CIME and CAM tags), if I have a CESM run crash (e.g. by segfaulting), I'm no longer seeing a message printing whether the run failed, with the log file location. The last stdout message is simply:

```
Mon Aug  3 18:38:55 2015 CESM EXECUTION BEGINS HERE
Mon Aug  3 18:48:05 2015 CESM EXECUTION HAS FINISHED
```

(To make this message easier to find, it should be in stdout rather than stderr I think, but for the record there's nothing printed to stderr, either.)

I see that the batch script still has Perl logic to do this, including this line:

```
appendCaseStatus("Model did not complete - see $config{'RUNDIR'}/$cesmlogfile");
```

However, there must be something wrong with the logic used to reach this point, since the log file exists, but nothing is printed.

**Comment by bandre-ucar** [comment]
I've had the same problem with segfaults not showing up in the log correctly. For what it's worth, my branch is based off of version d1b03ad (not blaming Bill, just demonstrating that the the problem goes back at least to mid-June).

It would be nice if the error message was dumped to both stdout and stderr.

**Comment by sholly** [comment]
fixed in #307
